staging/src/k8s.io/apiserver/pkg/server/config.go

```diff
@@ -842,6 +868,20 @@ func (c completedConfig) New(name string, delegationTarget DelegationTarget) (*G
 		}
 	}
 
+	// Add PostStartHooks for Unknown Version Proxy filter.
+	if c.PeerProxy != nil {
```

Comment:
I expected this in kube-apiserver or kube-aggregator, not here. Was that considered?